src/Compiler/Driver/OptimizeInputs.fs

```diff
@@ -48,6 +50,259 @@ let GetInitialOptimizationEnv (tcImports: TcImports, tcGlobals: TcGlobals) =
     let optEnv = List.fold (AddExternalCcuToOptimizationEnv tcGlobals) optEnv ccuinfos
     optEnv
 
+/// Result of the 'FirstLoop' phase
```

Comment:
Not for this PR, but we should consider splitting optimizer to multiple files, it's only getting bigger.